scss/_patterns_section.scss

```diff
@@ -7,6 +7,11 @@
     @media (min-width: $breakpoint-large) {
       padding-bottom: $spv--strip-regular;
     }
+
+    // Signpost is used with an H1 to its right, so we need to align top of the image to the top of the H1
+    .p-section__signpost-image {
```

Comment:
If there is any CSS needed for this it would not be part of section component.

"Signpost" are purely part of hero, and don't have much use outside of it as far as I understand.
We do have examples with signpost images already in Vanilla without any additional styling, so not sure why it's needed here.
https://vanillaframework.io/docs/examples/patterns/hero/hero-signpost

Reply:
The separate styling comes from the padding shown here in Figma:
![image](https://github.com/user-attachments/assets/6ee50c92-531f-4647-81aa-8a10dbeb868c)

With the padding:
![image](https://github.com/user-attachments/assets/52ec2589-2d64-404d-bce3-38be53ff61bb)

Without:
![image](https://github.com/user-attachments/assets/911938f0-b83e-4513-9988-fe71ab007791)

Existing Signpost logo example does not have this padding so the logo and the heading are not aligned. 

@lyubomir-popov Is this new padding correct?

Reply:
Whatever this padding is, it should not be part of hero pattern. 

If we need something to align images, it needs to be in image component. Some images will have whitespace on top (like many of our logos have), so setting any arbitrary top padding may depend on the image in question.